### PR TITLE
[RORO-4899][BT] make status tags lowercase

### PR DESCRIPTION
**Pull request by tipperba**

### CODE REVIEW

**Review by elabeca — DISMISSED** [review]

File: conf/messages [hunk 1]
```diff
@@ -23,6 +23,6 @@ dashboard.unknown.p1=The Check GVMS availability service is not working right no
 dashboard.unknown.p2=You can log into the {0} to check this yourself.
 dashboard.gvms.link_text=Goods Vehicle Movement Service (GVMS)
 
-service.state.AVAILABLE=AVAILABLE
-service.state.UNAVAILABLE=KNOWN ISSUES
-service.state.UNKNOWN=UNKNOWN
+service.state.AVAILABLE=Available
+service.state.UNAVAILABLE=Known Issues
```
Comment (sureshhmrc) [inline conf/messages:27]: i think its lower case `issues` now ?

File: conf/messages.cy [hunk 2]
```diff
@@ -1,6 +1,6 @@
 service.name = customs-service-status-frontend
 service.text = Dyma'ch gwasanaeth newydd
 
-service.state.AVAILABLE=AVAILABLE
-service.state.UNAVAILABLE=UNAVAILABLE
-service.state.UNKNOWN=UNKNOWN
+service.state.AVAILABLE=Available
+service.state.UNAVAILABLE=Unavailable
```
Comment (sureshhmrc) [inline conf/messages.cy:5]: should be `Known issues`

**Review by sureshhmrc — DISMISSED** [review]

File: conf/messages (hunk 1 above)
Comment (tipperba) [reply conf/messages]: corrected

File: conf/messages.cy (hunk 2 above)
Comment (tipperba) [reply conf/messages.cy]: this is being replaced by the Welsh, but have corrected.

**Review by sureshhmrc — APPROVED** [review]

**Review by sureshhmrc — APPROVED** [review]

**Pull request merged**